samples/BlazorServerSide/Pages/_Host.cshtml

```diff
@@ -21,6 +21,7 @@
     </environment>
     <link href="_content/Blazored.Typeahead/blazored-typeahead.css" rel="stylesheet" />
     <link href="css/site.css" rel="stylesheet" />
+    <link href="_content/Blazored.Typeahead/blazored-typeahead.css" rel="stylesheet" />
 </head>
 <body>
     <app>@(await Html.RenderComponentAsync<App>(RenderMode.Server))</app>
```

Comment:
Can this be updated to the new component tag helper?
```suggestion
    <app><component type="typeof(App)" render-mode="ServerPrerendered" /></app>
```

Reply:
Completed & pushed